processes/release_processes/README.MD

```diff
@@ -12,14 +12,42 @@ If you're the lucky "Release Rotation Engineer" this time, you should do the fol
 First: Run the publish script to create a new version of Cromwell.
 Second: Run through the "How to Release Cromwell into Firecloud" process.
 Third: Run through the "How to Deploy Cromwell in CAAS prod" process.
- 
+
+### How to publish a new Cromwell version
+
+#### Get a Github token
+
+The release WDL uses a github token to perform actions on your behalf.
+* Follow the directions [here](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
+    * When asked, give the token the full set of `repo` permissions. This will allow it to manipulate the Cromwell repository on your behalf.
+* Copy the token and save it to a file on your local machine. You'll use it in a second.
+
+#### Run the `publish_workflow.wdl` Workflow
+
+* Clone the Cromwell repo somewhere local.
+* Create an inputs json that looks something like this:
+
+```json
+{
+  "publish_workflow.githubTokenFile": "<<PATH TO TOKEN FILE>>",
+  "publish_workflow.majorRelease": true,
+  "publish_workflow.publishHomebrew": true,
+  "publish_workflow.publishDocker": "broadinstitute/cromwell-publish:latest",
+  "publish_workflow.organization": "broadinstitute"
+}
+```
+
+* Run the Cromwell instance using the local-backend.
```

Comment:
Closing other running memory-heavy applications is probably also a good idea (it's memory hungry!)